src/components/UserSelector/UserSelector.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+
+import { User } from '../../types/User';
+import { ActiveUserContext } from '../../utils/Store';
+import { DropdownMenu } from './DropdownMenu';
+
+interface UserSelectorProps {
+  users: User[];
+}
+
+export const UserSelector: React.FC<UserSelectorProps> = ({ users }) => {
+  const [isActive, setIsActive] = useState(false);
+  const { activeUser } = useContext(ActiveUserContext);
+
+  return (
+    <div
+      data-cy="UserSelector"
+      className={classNames('dropdown', { 'is-active': isActive })}
+    >
+      <div className="dropdown-trigger">
+        <button
+          tabIndex={0}
+          type="button"
+          className="button"
+          aria-haspopup="true"
+          aria-controls="dropdown-menu"
+          onClick={() => (isActive ? setIsActive(false) : setIsActive(true))}
+          onBlur={event => {
```

Comment:
Consider renaming the event handler function to follow the naming conventions for event handler functions. For example, you could use 'handleBlur'.